specs/models/fizzbuzzer_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rspec'
+require_relative "../../models/fizzbuzzer"
+
+describe Fizzbuzzer do
```

Comment:
You should be able to rethink this whole spec suit when the class only checks the value and does not iterate